pkg/controllers/raycluster_webhook.go

```diff
@@ -253,7 +253,7 @@ func oauthProxyContainer(rayCluster *rayv1.RayCluster) corev1.Container {
 			"--tls-cert=/etc/tls/private/tls.crt",
 			"--tls-key=/etc/tls/private/tls.key",
 			"--cookie-secret=$(COOKIE_SECRET)",
-			"--openshift-delegate-urls={\"/\":{\"resource\":\"pods\",\"namespace\":\"default\",\"verb\":\"get\"}}",
+			"--openshift-delegate-urls={\"/\":{\"resource\":\"pods\",\"namespace\":\"" + rayCluster.Namespace + "\",\"verb\":\"get\"}}",
```

Comment:
is this defaulted somewhere? I took a quick look and didn't wasn't sure where it's configured. Not sure how that will affect the behavior but at least wanted to ask.

Reply:
Is what defaulted?

Reply:
Where is the default value for `rayCluster.Namespace` set?

Reply:
Ahh, there is no default value for rayCluster.Namespace. It is a namespaced CR so it should always have a Name and a Namespace

Reply:
Cool, that makes sense! Just wanted to be sure. :)